src/queries/rum-dashboard.sql

```diff
@@ -1,5 +1,6 @@
 --- description: Get Helix RUM data for a given domain or owner/repo combination
 --- Authorization: none
+--- Access-Control-Allow-Origin: *
```

Comment:
@dylandepass any comment like that (starting with `---` and a capital letter) at the beginning of the query will be passed as a HTTP response header. This means you can add the CORS headers to the queries that need them yourself.